optbinning/binning/continuous_binning.py

```diff
@@ -715,10 +716,17 @@ def _fit(self, x, y, sample_weight, check_input):
             min_x = None
             max_x = None
 
+        if self.dtype == "numerical":
+            monotonic_score_kendalltau, kendalltau_pvalue = kendalltau(x_clean, y_clean)
```

Comment:
I find a bit confusing referring to the Kendal tau or Spearman rho correlation as a monotonic score. I think is best to choose one and call it correlation.

Reply:
U r right !